.golangci.yml

```diff
@@ -0,0 +1,61 @@
+run:
+  timeout: 5m
+  modules-download-mode: readonly
+
+linters-settings:
+  goconst:
+    min-len: 2
+    min-occurrences: 2
+  gofmt:
+    simplify: true
+  goimports:
+    local-prefixes: github.com/mattermost/mattermost-starter-template
```

Comment:
Will modifying this file be a pain when this repository is used as a template? Can we automate updating this?

Reply:
We could add a "What do I have to do after I forked this Repo" section to the `README`, which explains that this line has to be changed.

If you agree, I would add this as a separate PR